src/cleanlab_codex/response_validation.py

```diff
@@ -288,9 +296,9 @@ def is_unhelpful_response(
         f"{question}"
     )
 
-    output = tlm.prompt(prompt, constrain_outputs=["Yes", "No"])
-    response_marked_unhelpful = output["response"].lower() == expected_unhelpful_response
-    is_trustworthy = trustworthiness_score_threshold is None or (
-        output["trustworthiness_score"] > trustworthiness_score_threshold
-    )
-    return response_marked_unhelpful and is_trustworthy
+    output = tlm.get_trustworthiness_score(prompt, response=expected_unhelpful_response, constrain_outputs=["Yes", "No"])
```

Comment:
for this use-case, did you find this works better than:

```
tlm.get_trustworthiness_score(prompt_that_includes_unsure_option, response=expected_unhelpful_response, constrain_outputs=["Yes", "No", "Unsure"]) 
```

Reply:
I'd guess just Yes/No probably does work better for this use-case

Reply:
Just Yes/No works better for this use case.